public/index.html

```diff
@@ -6,12 +6,20 @@
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
     <link rel="icon" href="<%= BASE_URL %>favicon.ico">
     <title><%= htmlWebpackPlugin.options.title %></title>
+    <script src="https://kit.fontawesome.com/b21073efe8.js" crossorigin="anonymous"></script>
```

Comment:
you can install fontawesome using npm

`npm install --save @fortawesome/fontawesome-free`